plugins/pulp_rpm/plugins/importers/yum/sync.py

```diff
@@ -845,13 +849,10 @@ def _concatenate_units(self, existing_unit, new_unit):
                                              (existing_unit.unit_key, new_unit.unit_key))
 
         if isinstance(existing_unit, models.Errata):
-            # add in anything from new_unit that we don't already have. We key
-            # package lists by name for this concatenation.
-            existing_package_list_names = [p['name'] for p in existing_unit.pkglist]
-
-            for possible_new_pkglist in new_unit.pkglist:
-                if possible_new_pkglist['name'] not in existing_package_list_names:
-                    existing_unit.pkglist += [possible_new_pkglist]
+            # Add in collectons from new_unit pkglist or update existing_unit pkglist collection
```

Comment:
This comment is about a behavior in `merge_errata` so moving it to that docstring would be good.
s/collectons/collections/